AutomatedTesting/Gem/PythonTests/assetpipeline/ap_fixtures/clear_moveoutput_fixture.py

```diff
@@ -4,7 +4,7 @@
 
 SPDX-License-Identifier: Apache-2.0 OR MIT
 
-Fixture for clearing out 'MoveOutput' folders from \dev and \dev\PROJECT
+Fixture for clearing out 'MoveOutput' folders from \\dev and \\dev\\PROJECT
```

Comment:
Actually this comment is incorrect in the first place.

We don't refer to <dev> anymore as it doesn't have a defined meaning.

We refer to <engine-root> and <project-root>, since they have a known definition and it doesn't apply that the project is located within a dev folder

Reply:
Should I just remove this comment? Or change `dev` to `ENGINE_ROOT `?

Reply:
I think you should remove the comment.

Reply:
Fixed.